src/main/resources/data.sql

```diff
@@ -18,7 +18,7 @@ create table address (
 create table payment (
     id varchar(60) primary key,
     user_id varchar(60) not null,
-    card_number varchar(16) not null unique,
+    card_number varchar(16) not null, -- No need to be unique (or only support payments by russian hackers using different card every time?)
```

Comment:
varchar allows for characters.  Is that the right choice here?

Reply:
I think so.  The credit card 'number' is really a string consisting of numbers, you wouldn't want it to be treated like an actual number at any point, and you may do things to obscure part of the string, etc.

Reply:
Thinking of it as a string of numbers is somewhere my mind didn't go initially.  I like it :) .  There are some numerical checks on credit cards like something that starts with a 4 is Visa and checks where the values can't be transposed.  That seems possible to determine if it is a char or a number so not an issue unless we are squeezing performance.  A number would likely save some DB space and a bit of memory as it would be able to be an unsigned number.